skimage/draw/_random_shapes.py

```diff
@@ -312,16 +344,28 @@ def random_shapes(image_shape,
                     msg = 'Intensity range must lie within (0, 255) interval'
                     raise ValueError(msg)
 
+    if background is None:
+        background = [255] * num_channels
+    elif isinstance(background, int):
+        background = [background]
```

Comment:
[background] * num_channels?

Reply:
Also, what about the case where the user wants to specify RGB background.

Reply:
```[background] * num_channels```: good idea. My initial idea was to not allow single values for background in multichannel images. But this is more convenient.

I'm not shure if I understand you correctly regarding RGB background. The user can specify background for images of three channels with a tuple of length 3. Or do you think of matplotlib-like color strings as e.g.```background='r'``` for red background?